app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+---
+title: Encapsulated style with Polymer 2.0 and shadow DOM v1
+---
+
+<!-- toc -->
+
+## Styling your elements
+
+Polymer supports DOM templating and the shadow DOM API. When you use the `<dom-module>` and `<template>` blocks to template your custom element, Polymer attaches a shadow root to instances of this element. Polymer then copies in the contents of the template you provided for your element.
+	
+Here's an example:
+
+[See it on Plunker](http://plnkr.co/edit/VPktVlI0PWMnYdYVsVBx?p=preview)
+
+```html
+<!-- Create a template for the custom element -->
+<dom-module id='custom-element'>
```

Comment:
Random point of style here: our convention is to use double quotes in HTML (id="custom-element") and single quotes in JSON unless there's a compelling reason to do otherwise.